spec/acceptance/rundeck_spec.rb

```diff
@@ -7,9 +7,7 @@
       class { 'java':
         distribution => 'jre'
       }
-      class { 'rundeck':
-        package_ensure => '2.11.5'
-      }
+      class { 'rundeck': }
```

Comment:
I think it would be nice to run ne acceptance test with 2.11.5 and afterwards check if an upgrade to 3.x works.

Reply:
docs are, uhm, rare for beaker :(. it basically executes the puppet code in the spec/acceptance/rundeck_spec.rb file from top to bottom. There are already to blocks with rundeck 2.11.5. They will be executed in the same docker container. You can add another block at the bottom with the latest rundeck version, then puppet will upgrade from 2.11.5 to 3.